google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/vcs/GcpCheckoutProvider.java

```diff
@@ -186,7 +186,8 @@ private static boolean doClone(@NotNull Project project, @NotNull ProgressIndica
       return true;
     }
     VcsNotifier.getInstance(project)
-        .notifyError(GctBundle.message("clonefromgcp.failed"), result.getErrorOutputAsHtmlString());
+        .notifyError(GctBundle.message("clonefromgcp.failed"), result.getErrorOutputAsHtmlString()
+            + " " + result.getOutputAsJoinedString());
```

Comment:
one thing I noticed, you may be able to use this overloaded method instead and provide a listener for the URL clicks:

`notifyError(@NotNull String title, @NotNull String message, @Nullable NotificationListener listener)`